src/Maestro/Microsoft.DotNet.Maestro.Tasks.Tests/TestManifests/UnversionedBlob/UnversionedBlob.xml

```diff
@@ -0,0 +1,111 @@
+<Build PublishingVersion="3" Name="dotnet-arcade" BuildId="20201016.5" Branch="refs/heads/master" Commit="e7a79ce64f0703c231e6da88b5279dd0bf681b3d" AzureDevOpsAccount="dnceng" AzureDevOpsBranch="refs/heads/master" AzureDevOpsBuildDefinitionId="6" AzureDevOpsBuildId="856354" AzureDevOpsBuildNumber="20201016.5" AzureDevOpsProject="internal" AzureDevOpsRepository="https://dnceng@dev.azure.com/dnceng/internal/_git/dotnet-arcade" InitialAssetsLocation="https://dev.azure.com/dnceng/internal/_apis/build/builds/856354/artifacts" IsReleaseOnlyPackageVersion="False" IsStable="False">
```

Comment:
You can probably scope down the elements in this XML to only represent the values that you need. Then, your XML should be small enough you could turn it into a string like the others I suggested and not need this file.

Reply:
Refactor took care of the files.